File: Beginner/1072 (Intervals 2)/1072.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            int acmIn = 0;
            int acmOut = 0;
         

            for (int i = 0; i < n; i++)
            {
                int x = int.Parse(Console.ReadLine());
                if(x >= 10 && x <= 20)
                {
                    acmIn++;
                }
                else
                {
                    acmOut++;
                }
               
            }

            Console.WriteLine(acmIn + " in");
            Console.WriteLine(acmOut + " out");

        }
    }
}

File: Beginner/1072 (Intervals 2)/1072.py
n = int(input())
acmIn = int()
acmOut = int()

for i in range(0,n):
    x = int(input())
    if x >= 10 and x <= 20:
        acmIn += 1
    else:
        acmOut += 1

print(acmIn, "in")
print(acmOut, "out")